go/libraries/doltcore/merge/conflict_source.go

```diff
@@ -47,7 +47,10 @@ type ConflictReader struct {
 // NewConflictReader returns a new conflict reader for a given table
 func NewConflictReader(ctx context.Context, tbl *doltdb.Table) (*ConflictReader, error) {
 	base, sch, mergeSch, err := tbl.GetConflictSchemas(ctx)
-	if err == doltdb.ErrNoConflicts {
+	if err != nil {
```

Comment:
Callers of this method are still checking for ErrNoConflicts

Need to delete that error type entirely to make sure you have all references

Reply:
I think I removed all references to this error since I deleted `ErrNoConflicts` from `go/libraries/doltcore/doltdb/errors.go`